### DISCUSSION TITLE
File Upload in chuck missing guide on page - content/guides/http/direct-file-uploads.md

### DISCUSSION THREAD

**Discussion by bhargavfortius** [adonisjs/core#4163] [Help]
How to upload chunk using s3 
in this section not mention someSDK 
```
request.multipart.onFile('input_field_name', {}, (part) => {
  someSdk.uploadStream(part)
})

await request.multipart.process()

const file = request.input('input_field_name')
if (file.hasErrors) {
  return file.errors
}
```
please mention example for aws s3 chunk upload on docs in on this part "content/guides/http/direct-file-uploads.md"

@RomainLanz

**Comment by RomainLanz** [comment]
Hey @bhargavfortius! 👋🏻 

While we strive to provide comprehensive documentation, it's not feasible for us to cover every SDK in the world.
However, we recommend checking out the [AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpu-upload-object.html).

**Reply by bhargavfortius** [reply]
Okay, but please ask for your docs to provide a way. if aws side needs then I am not msg in your side because aws gave proper docs for aws docs and Stackoverflow
